src/analysis/windowed_indicators.jl

```diff
@@ -130,18 +215,90 @@ function estimate_indicator_changes(config::WindowedIndicatorConfig, x, t = each
     end
 
     # put everything together in the output type
-    return WindowedIndicatorResults(
+    return SlidingWindowResults(
         t, x, t_indicator, x_indicator, t_change, x_change, config
     )
 end
 
+"""
+    estimate_indicator_changes(config::SegmentWindowConfig, x [,t]) → output
```

Comment:
This function should NOT have two documentation strings. The function does _the same_ thing. It performs the estimation of indicators and change metrics as instructed by the configuration struct. Hence, we only need one simple docstring that shouldn't be more than 2 or 3 lines of text. The main explanation is happening in the configuration structs. The configuration strucs already say that they are passed into `estimate_indicator_cjhanges`.